packages/dataviews/README.md

```diff
@@ -150,25 +150,16 @@ Example:
 [
 	{
 		id: 'date',
+		type: 'date',
 		header: __( 'Date' ),
 		getValue: ( { item } ) => item.date,
```

Comment:
This could be optional too because it's just `item[ id ]`

Reply:
I think it's good to keep them both because it's just a coincidence that we the `id` here has the same `type`. This kind of shortcut would work for just a handful of cases(only the field types we support).